### PR TITLE
I5520 ip tables

### PR DESCRIPTION
**Pull request by kayiwa**
use a iptables template to set firewall rules.
the order of the in your configuration matter

### CODE REVIEW

File: roles/ufw_firewall/handlers/main.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 ---
 # handlers file for roles/ufw_firewall
-- name: restart ufw
-  ansible.builtin.service:
-    name: ufw
-    state: restarted
+- name: Reload UFW
+  ansible.builtin.command: ufw reload
+  changed_when: false
```
Comment (acozine) [inline roles/ufw_firewall/handlers/main.yml:5]:
```suggestion
```

**Review by acozine — COMMENTED** [review]
Oops, I left this in draft form. . . .

File: group_vars/abid/staging.yml [hunk 2]
```diff
@@ -1,23 +1,41 @@
...
-      - 172.20.95.0/24
-      - 172.20.192.0/19
-  - service: http
+    action: ACCEPT
+  - protocol: tcp
+    source: 10.249.0.0/18
+    port: 22
+    action: ACCEPT
+  - protocol: tcp
+    source: 128.112.0.0/16
```
[the remark this answers is not in the document]
Comment (kayiwa) [reply group_vars/abid/staging.yml:14]: Further discussion can [happen here](https://github.com/pulibrary/princeton_ansible/pull/5500)

**Comment by acozine** [comment]
My remaining review comments will be addressed in the next PR. This is a step forward.

**Pull request merged**